common/content/statusline.js

```diff
@@ -392,6 +392,32 @@ const StatusLine = Module("statusline", {
 
                 node.value = bufferPositionStr;
             });
+        statusline.addField("zoomlevel", "The main content's zoom level", "liberator-status-zoomlevel",
+            /**
+             * Display the main content's zoom level.
+             *
+             * @param {Element} node
+             * @param {number} percent The position, as a percentage. @optional
+             * @param {boolean} full True if full zoom is in operation. @optional
+             */
+            function updateZoomLevel (node, percent, full) {
+                if (typeof(buffer) == "undefined")
+                    return;
+
+                if (!percent || typeof percent != "number") {
+                    percent = buffer.zoomLevel;
+                }
+                
+                if (percent == 100)
+                    node.value = "";
+                else {
+                    percent = ("  " + Math.round(percent)).substr(-3);
+                    if (full)
+                        node.value = " [" + percent + "%]";
```

Comment:
I don't think statusline items should include padding themselves. Preferably the leading space would be removed, i.e. `"["` and `"("` vs `" ["` and `" ("`.